deb-get

```diff
@@ -532,6 +532,17 @@ function deb_quickemu() {
     SUMMARY="Quickly create and run optimised Windows, macOS and Linux desktop virtual machines."
 }
 
+function deb_virtualbox-6.1() {
+    ARCHS_SUPPORTED="amd64"
+    APT_KEY_URL="https://www.virtualbox.org/download/oracle_vbox_2016.asc"
+    APT_LIST_NAME="virtualbox-6.1"
+    APT_REPO_URL="deb [arch=amd64] https://download.virtualbox.org/virtualbox/debian ${OS_CODENAME} contrib"
```

Comment:
I'm pretty sure your are supposed to use `UPSTREAM_CODENAME` instead.

Reply:
I did, but was having failures with some alien distros, Looking at it again now.